docs/users/cli-usage.md

```diff
@@ -374,15 +375,23 @@ Commands can also be executed using the shorter `ddev . <cmd>` alias.
 The `ddev ssh` command will open an interactive bash shell session to the container for a ddev service. The web service is connected to by default. The session can be ended by typing `exit`. To connect to another service, use the `--service` flag to specify the service you want to connect to. For example, to connect to the database container, you would run `ddev ssh --service db`.
 
 ### Log Access
+
 The `ddev logs` command allows you to easily retrieve error logs from the web server. To follow the web server error log (watch the lines in real time), run `ddev logs -f`. When you are done, press CTRL+C to exit from the log trail.
 
-Additional logging can be accessed by using `ddev ssh` to manually review the log files you are after. The web server stores access logs at `/var/log/nginx/access.log`, and PHP-FPM logs at `/var/log/php7.0-fpm.log`.
+To manually review additional log files use the `ddev ssh` command. The web server stores access logs at `/var/log/nginx/access.log`, and PHP-FPM logs at `/var/log/php7.0-fpm.log`.
 
 ## Stopping a project
-You can stop a project's containers without losing data by using `ddev stop` in the working directory of the project. You can also stop any running project's containers by providing the project name as an argument, e.g. `ddev stop <projectname>`.
+
+Stop a project's containers without losing data by using `ddev stop` in the working directory of the project. Stop any running project's containers by providing the project name as an argument, e.g. `ddev stop <projectname>`.
 
 ## Removing a project
-You can remove a project's containers by running `ddev remove` in the working directory of the project. You can also remove any running project's containers by providing the project name as an argument, e.g. `ddev remove <projectname>`. `ddev remove` is *not* destructive. It removes the docker containers but does not remove the database for the project, so you can easily have many configured projects with databases loaded, but with no docker containers wasted on unused projects. Your project code base and files are never affected by `ddev remove`. When you want to remove the imported database for a project, use `ddev remove --remove-data` instead of just `ddev remove`, and the database files will also be destroyed.
+
+To remove a project's containers run `ddev remove` in the working directory of the project. To remove any running project's containers, providing the project name as an argument, e.g. `ddev remove <projectname>`.
+
+`ddev remove` is *not* destructive. It removes the docker containers but does not remove the database for the project. This allows you to have many configured projects with databases loaded without wasting docker containers on unused projects. `ddev remove` does not affect the project code base and files.
+
+To remove the imported database for a project, use `ddev remove --remove-data` instead of just `ddev remove`, and the database files will also be destroyed.
```

Comment:
"and both the containers and the imported database contents will be destroyed"

Reply:
Fixing, but it also made me realize I let some passive language survive. 

Hmm... I questioning this - if the commands should be on their own lines. 

> To remove the imported database for a project, use the flag `--remove-data`, as in `ddev remove --remove-data`. This command will destroy both the containers and the imported database contents.